src/utils/LoggingHelper.ts

```diff
@@ -46,4 +49,27 @@ export default class LoggingHelper {
       siteID: siteArea.siteID,
     };
   }
+
+  public static getCarProperties(car: Car): { carID: string; userID: string, carCatalogID: number } {
+    return {
+      carID: car.id,
```

Comment:
Do both carID and carCatalogID are stored in the logs?
This is the purpose of this class.